CMSgov/mdct-mcr
# Mdct 2125

## Description
modal drawer content for pdf 

### How to test
When there are no entity cards, the pdf should have text saying that the user did not fill out any of that data.
When there are entity cards that aren't completed: If a user doesn't fill out the second half, each question should feature "Not Answered" in red.  The image on the left should be red with an 'error!' underneath.
When there are entity cards that are completed: The card should have a green image with "Complete' underneath. All data field should display the question and answer.
Indicators (e.g. C2.V.I) should only be displayed in the pdf view.


### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
